gmachine.go

```diff
@@ -93,6 +95,19 @@ func (g *GMachine) Run() {
 				}
 				fmt.Fprintf(g.Stderr, "%c", g.A)
 			}
+		case CMP:
+			value := g.Memory[g.P]
```

Comment:
It's now so common to fetch the next memory location and bump the program counter that you might implement this as something like `g.Next()`.